docs/source/manual/hibernate.rst

```diff
@@ -125,7 +125,7 @@ contains type-safe wrappers for most of ``SessionFactory``'s common operations:
 
         @SuppressWarnings("unchecked")
```

Comment:
You should be able to remove this now since you're no longer casting a generic type

Reply:
Thank you for pointing that out! I've adjusted the comment and removed the `SuppressWarnings` annotation.